### PR TITLE
chore(docs): updating deprecated huggingface-cli to hf

### PR DESCRIPTION
**Pull request by Maximellerbach**
chore(docs): updating deprecated huggingface-cli to hf
## Type / Scope

- **Type**:  Chore
- **Scope**: docs

## Summary / Motivation

- Removed any occurences of the deprecated `huggingface-cli` in the docs.

## Related issues

- Fixes / Closes: # (if any)
- Related: # (if any)

## What changed

- replaced `huggingface-cli` with `hf`

## How was this tested (or how to run locally)

- manually checked the commands
- didn't test to upload a valid repo env, but created an empty repo (should work)

## Checklist (required before merge)

- [x] Linting/formatting run (`pre-commit run -a`)
- [x] All tests pass locally (`pytest`)
- [x] Documentation updated
- [x] CI is green

## Reviewer notes

- Anything the reviewer should focus on (performance, edge-cases, specific files) or general notes.
- Anyone in the community is free to review the PR.

### CODE REVIEW

File: docs/source/envhub.mdx [hunk 1]
```diff
@@ -155,10 +155,10 @@ Upload your repository to Hugging Face:
 pip install huggingface_hub
 
 # Login to Hugging Face
-huggingface-cli login
+hf auth login
 
 # Create a new repository
-huggingface-cli repo create my-custom-env --type space --org my-org
+hf repo create my_org/my-custom-env
```
[the remark this answers is not in the document]
Comment (Maximellerbach) [reply docs/source/envhub.mdx:162]: the repo type for env for now is models (which is the default type when using `hf repo create`)

File: docs/source/lekiwi.mdx [hunk 2]
```diff
@@ -279,13 +279,13 @@ We use the Hugging Face hub features for uploading your dataset. If you haven't
...
 ```bash
-huggingface-cli login --token ${HUGGINGFACE_TOKEN} --add-to-git-credential
+hf auth login --token ${HUGGINGFACE_TOKEN} --add-to-git-credential
 ```
 
 Then store your Hugging Face repository name in a variable:
 
 ```bash
-HF_USER=$(huggingface-cli whoami | head -n 1)
+HF_USER=$(hf auth whoami | awk -F': *' 'NR==1 {print $2}')
```
Comment (jadechoghari) [inline docs/source/lekiwi.mdx:288]: if you're sure adding print $2 is more robust, feel free to close this
